src/index.js

```diff
@@ -10,16 +10,17 @@ var Blocks = require('./engine/blocks');
  * Handles connections between blocks, stage, and extensions.
  *
  * @author Andrew Sliwinski <ascii@media.mit.edu>
+ * @param {!RenderWebGL} renderer Renderer for the VM
  */
-function VirtualMachine () {
+function VirtualMachine (renderer) {
```

Comment:
Would maybe prefer it I think if this were an optional parameter (`opt_renderer`, `@param {RenderWebGL=}`) or if it was set after instantiation (`vm.attachRenderer`)? I'm imagining we might sometimes want a VM with no renderer.